dropwizard-scala-parent-pom/pom.xml

```diff
@@ -0,0 +1,218 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.mainstreethub</groupId>
+    <artifactId>dropwizard-parent-pom</artifactId>
+    <version>1.0.2-4</version>
+  </parent>
+
+  <artifactId>dropwizard-scala-parent-pom</artifactId>
+  <version>1.0.2-1-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <scm>
+    <url>https://github.com/mainstreethub/parent-poms</url>
+    <connection>scm:git:git@github.com:mainstreethub/parent-poms</connection>
+    <developerConnection>scm:git:git@github.com:mainstreethub/parent-poms</developerConnection>
+    <tag>HEAD</tag>
+  </scm>
+
+  <properties>
+    <dropwizard-scala.commons.version>0.0.1</dropwizard-scala.commons.version>
+    <dropwizard-scala.version>1.0.0-2</dropwizard-scala.version>
+    <dropwizard.test.version>1.0.0</dropwizard.test.version>
+    <json4s.version>3.5.0</json4s.version>
+    <mysql-connector.version>5.1.38</mysql-connector.version>
+    <mysql-junit4.version>0.2</mysql-junit4.version>
+    <okhttp.version>3.3.0</okhttp.version>
+    <scala_major.version>2.11</scala_major.version>
+    <scala.version>${scala_major.version}.8</scala.version>
```

Comment:
I did some googling, but I didn't find a good way to parse a variable so I could reference it in the dependencies section.  It could also be that I haven't used maven much and this is an easy thing to do that I can't figure out.  I could split it into three parts ${major}.${minor}.${incremental} but I'm not sure that's any less brittle